app/src/main/java/org/fossasia/openevent/general/event/EventUtils.kt

```diff
@@ -63,4 +63,26 @@ object EventUtils {
             }
         return testDate.time
     }
+
+    fun getFormattedDate(date: ZonedDateTime): String{
+        lateinit var dateString : String
+        val dateFormat: DateTimeFormatter = DateTimeFormatter.ofPattern("EEE, MMM d")
+        try {
+            dateString = dateFormat.format(date)
+        } catch (e: IllegalArgumentException) {
+            Timber.e(e,"Error formatting Date")
+        }
+        return dateString
```

Comment:
This will throw exception if used in case of exception

Reply:
should i remove try catch then

Reply:
No, send empty string if there's an exception

Reply:
okay